src/app/modules/item/pages/item-edit/item-edit.component.html

```diff
@@ -22,7 +22,12 @@
       </nav>
 
       <div class="bg-white" >
-        <alg-item-edit-content *ngIf="rla1.isActive" [parentForm]="itemForm"></alg-item-edit-content>
+        <alg-item-edit-content
+          *ngIf="rla1.isActive"
+          [parentForm]="itemForm"
+          [itemData$]="itemData$"
+        >
```

Comment:
Typically we do not pass observable but just change the value when it changes (which may be considered as valid as well, but at least that's not what we do). Check how it is done in `item-details.component.html`.

Reply:
Done